phpstan-baseline.neon

```diff
@@ -1820,6 +1820,12 @@ parameters:
 			count: 1
 			path: src/Core/Checkout/Customer/Rule/CustomerCustomFieldRule.php
 
+		-
+			message: "#^Method Shopware\\\\Core\\\\Checkout\\\\Customer\\\\Rule\\\\CustomerCustomFieldRule\\:\\:isSwitchOrBoolField\\(\\) has parameter \\$renderedField with no value type specified in iterable type array\\.$#"
```

Comment:
Can you solve this instead of adding a new entry here?

Reply:
@shyim sure will handle that but that will require much more changes not only in this concrete class

Reply:
Not exactly. Just add phpdoc:
```php
/**
 * @param array<string, string> $renderedField
 */
```

Reply:
@dneustadt indeed that worked :) I use `@param array{type: string} $renderedField` initially and that seems to cascade other checks but with  `@param array<string, string> $renderedField` seemd fine.

@shyim so I did it as you and @dneustadt suggested.